pyo3-macros-backend/src/from_pyobject.rs

```diff
@@ -61,6 +61,11 @@ impl<'a> Enum<'a> {
                 if maybe_ret.is_ok() {
                     return maybe_ret
                 }
+                if let Err(inner) = maybe_ret {
```

Comment:
This pair of `if ok() / if let Err` looks like a `match` could be more appropriate.